imports/plugins/core/revisions/server/methods.js

```diff
@@ -112,17 +133,17 @@ Meteor.methods({
     if (revisions) {
       for (const revision of revisions) {
         if (!revision.documentType || revision.documentType === "product") {
-          const oldDocument = Products.findOne(revision.documentId);
-          if (oldDocument && revision.documentData.handle !== oldDocument.handle) {
-            revision.documentData.changedHandleWas = oldDocument.handle;
-          }
-          const res = Products.update({
-            _id: revision.documentId
-          }, {
-            $set: revision.documentData
-          }, {
-            publish: true
-          });
+          previousDocuments.push(Products.findOne(revision.documentId));
```

Comment:
`previousDocuments` is showing up as not defined then running the linter. I believe this is causing the server error @mikemurray found while testing.

Reply:
@mikemurray @nnnnat Publishing error fixed, it was introduced after merging release 1.9